src/test/java/org/broadinstitute/hellbender/utils/bigquery/AvroFileReaderUnitTest.java

```diff
@@ -0,0 +1,40 @@
+package org.broadinstitute.hellbender.utils.bigquery;
+
+
+import org.apache.avro.generic.GenericRecord;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.apache.avro.Schema;
+
+import java.util.*;
+
+/**
+ * A class to test the functionality of {@link AvroFileReader}.
+ */
+public class AvroFileReaderUnitTest extends GATKBaseTest {
+
+    private static final String avroFileURI = "gs://broad-dsp-spec-ops/kcibul/acmg.35.chr20.1mb.cohort.extract.avro";
```

Comment:
a few things here

1. make this file much smaller (~ kilobytes), just take the first 100 rows or something
2. move it into GATK repo (@mmorgantaylor has done this for other tests)